scripts/health_metrics/code_coverage_file_list.json

```diff
@@ -118,6 +167,14 @@
       "\\.github/workflows/messaging\\.yml"
     ]
   },
+  {
+    "sdk": "mlmodeldownloader",
```

Comment:
Does code coverage work for Swift pods? Do you have an example report?

Reply:
Code coverage is not working for newly added pods in this PR. Code coverage is more about hard coding here.
https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/health-metrics-presubmit.yml#L20-L35
Will also update the file name to file_patterns.json to avoid confusion.